CanvasColorSpaceProposal.md

```diff
@@ -140,18 +140,23 @@ IDL
 <pre>
 typedef (Uint8ClampedArray or Uint16ClampedArray or Float32Array) ImageDataArray;
 
-[Constructor(unsigned long sw, unsigned long sh, optional CanvasColorSpace colorSpace = "legacy-srgb"),
- Constructor(ImageDataArray data, unsigned long sw, optional unsigned long sh, optional CanvasColorSpace colorSpace),
+dictionary ImageDataColorSettings {
+  CanvasColorSpace colorSpace = "srgb";
+  CanvasPixelFormat pixelFormat = "8-8-8-8";
```

Comment:
The types possible for ImageData do not map 1 to 1 with CanvasPixelFormat.  I think a new enum is necessary